source/opt/def_use_manager.h

```diff
@@ -106,13 +112,21 @@ class DefUseManager {
   // Erases the records that a given instruction uses its operand ids.
   void EraseUseRecordsOfOperandIds(const ir::Instruction* inst);
 
-  IdToDefMap id_to_def_;    // Mapping from ids to their definitions
-  IdToUsesMap id_to_uses_;  // Mapping from ids to their uses
+  const MessageConsumer& consumer_;  // Message consumer.
+  IdToDefMap id_to_def_;             // Mapping from ids to their definitions
+  IdToUsesMap id_to_uses_;           // Mapping from ids to their uses
   // Mapping from instructions to the ids used in the instructions generating
   // the result ids.
   InstToUsedIdsMap inst_to_used_ids_;
 };
 
+inline DefUseManager::DefUseManager(const MessageConsumer& consumer,
+                                    ir::Module* module)
+    : consumer_(consumer) {
+  (void)consumer_;
```

Comment:
It's needed since we only have two SPIRV_ASSERT using it. Will error out on release build b/c unused field. I moved this statement to where SPIRV_ASSERT uses consumer to make it more readable.